index.bs

```diff
@@ -444,25 +444,13 @@ using a third party payment service from within an iframe)
 the [=top-level browsing contexts=] suddenly becomes in a position
 to carry out a skimming attack against the [=browsing context=] that has [=gains focus|gained focus=].
 
-To mitigate this threat,
-[=sensor reading|readings=] of [=device sensor|sensors=] running in a [=top-level browsing contexts=]
-are not delivered in such cases.
-A [=security check=] is run before [=sensor readings=] are delivered to ensure that.
-
+To mitigate this threat, user agent can check if it [=can expose sensor readings=].
 
 <h4 id="visibility-state">Visibility State</h4>
 
-[=Sensor readings=] are only available
-in [=top-level browsing context|browsing contexts=] that are visible to the user,
-that is, whose [=steps to determine the visibility state|visibility state=]
+[=Sensor readings=] are only available in [=top-level browsing context|browsing contexts=],
+whose [=active document=]'s [=steps to determine the visibility state|visibility state=]
 is "visible".
-A [=security check=] is run before [=sensor readings=] are delivered to ensure that.
-
-Issue: certain use cases require sensors to have background access.
-Using a more complex {{PermissionDescriptor}}.
-(e.g. with a boolean `allowBackgroundUsage = false`; [=dictionary member=]),
-might be the solution to relax this restriction.
-
```

Comment:
How is this issue being addressed?

Reply:
@rwaldron If you want, I can remove it in separate PR. We already have several issues related to exposing sensors to service workers https://github.com/w3c/sensors/projects/5. Removed inline issue is redundant.

Reply:
Added issue back, will remove it in separate PR.